src/Components/App/App.js

```diff
@@ -46,9 +46,12 @@ class App extends Component {
           <Route path='/myreflections'>
             <ReflectionLibrary />
           </Route>
-          <Route path='/breathing/:feeling'>
-            <BreathingRoom/>
-          </Route>
+          <Route path='/breathing/:feeling' render={({match}) => {
```

Comment:
This is SO helpful! Glad we had the chance to address this issue with the page reloading and leaving our code lost without a mood prop.